### ISSUE TITLE
Preferences: Replace SettingsSidebar and SimpleSettingsPage

### ISSUE THREAD

**Issue by ryonakano** [elementary/monitor#390]
### Problem

We're currently using Granite.SettingsSidebar and Granite.SimpleSettingsPage for the preference view:

![Screenshot from 2024-10-12 10-18-29](https://github.com/user-attachments/assets/a9db4c77-f806-415a-a761-9d07b1ede6c2)

But these widgets were moved to Switchboard and deprecated in Granite 7 (see https://github.com/elementary/switchboard/issues/265) so we would see the deprecation warning when we port Monitor to GTK 4.

### Proposal

Replace SettingsSidebar and SimpleSettingsPage with another widget, maybe Gtk.StackSwitcher?

**Comment by stsdc** [comment]
I agree. This PreferencesView needs also a redesign and visual cleanup.
Also on the top left and bottom right corner we can see how content overflows the Popover container.
